scss/partials/_cards.scss

```diff
@@ -102,5 +111,10 @@
     .content-block__title {
       @extend .a;
     }
+    img {
+      outline: none;
+      opacity: 0.95;
```

Comment:
Just FYI, the space between the image and the author is likely to get flagged by design. 
![image](https://user-images.githubusercontent.com/5826750/134723677-dd5f02b6-8590-43fa-b964-8b41e84da3eb.png)

Reply:
Gotcha, I messed up here the date needs to go underneath the title.